docs/support-runbook.md

```diff
@@ -0,0 +1,50 @@
+# Support runbook
+
+This document describes possible ways in which the system may fail or malfunction, with instructions how to handle and recover from these scenarios.
+
+This runbook is supplementary to the [general 24/7 support runbook](https://intranet.torchbox.com/propositions/design-and-build-proposition/delivering-projects/dedicated-support-team/247-support-out-of-hours-runbook/), with details about project-specific actions which may be necessary for troubleshooting and restoring service.
```

Comment:
Could we create a redirect to this page on the intranet and link this to that instead? We often reorganise the intranet and this might break the URL here